src/ts_catalog/catalog.h

```diff
@@ -119,6 +119,7 @@ enum Anum_hypertable
 	Anum_hypertable_compression_state,
 	Anum_hypertable_compressed_hypertable_id,
 	Anum_hypertable_replication_factor,
+	Anum_hypertable_flags,
```

Comment:
Should we name this status similar to chunk.status?

Reply:
sure, let's rename it to status